[MAUI][Android] 2.0.0-rc6 crashes on PopAsync

**Describe the bug**
NavigationPage's PopAsync crashes on ChartViewHandler.DisconnectHandler()

- MAUI Shell not used
- XAML not used

Null here (LINE 71): https://github.com/beto-rodriguez/LiveCharts2/blob/53739c43090aa44b005876a9a2c882b0d8ef320c/src/_Shared.Native/Platforms/Android/PointerController.cs#L61-L73

When user does not touch the chart - it never gets _customScaleListener so it crashes on Dispose() call because of missing nullcheck.

https://github.com/beto-rodriguez/LiveCharts2/blob/53739c43090aa44b005876a9a2c882b0d8ef320c/src/_Shared.Native/Platforms/Android/PointerController.cs#L83-L93

Make it safe?
```cs
public void DisposeController(object view) 
{
...
_customScaleListener?.Dispose(); //missing ? for null check
```

**Expected behavior**
Should not crash

**Actual behavior**
Crashes on Page pop

**Steps to reproduce**
1. Download [TestRepo](https://github.com/Domik234/Issue-LiveCharts-Crashing)
2. Run app on Android
3. Navigate to ChartPage
4. Pop to previous page

<details>
<summary>GIF</summary>

![Image](https://github.com/user-attachments/assets/a69d1a48-47b7-46bf-8778-c3601d162afb)

</details>

**Smartphone (please complete the following information):**
 - Device: Honor 20
 - OS: 10
 
```text
**Additional context**
**System.NullReferenceException:** 'Object reference not set to an instance of an object.'
**System.NullReferenceException:** 'Object reference not set to an instance of an object.'
[mono-rt] [ERROR] FATAL UNHANDLED EXCEPTION: System.NullReferenceException: Object reference not set to an instance of an object.
[mono-rt]    at LiveChartsCore.Native.PointerController.DisposeController(Object view)
[mono-rt]    at LiveChartsCore.SkiaSharpView.Maui.Handlers.ChartViewHandler.DisconnectHandler(ContentViewGroup platformView)
[mono-rt]    at Microsoft.Maui.Handlers.ViewHandler`2[[Microsoft.Maui.IContentView, Microsoft.Maui, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null],[Microsoft.Maui.Platform.ContentViewGroup, Microsoft.Maui, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]].OnDisconnectHandler(View platformView) in /_/src/Core/src/Handlers/View/ViewHandlerOfT.cs:line 79
[mono-rt]    at Microsoft.Maui.Handlers.ViewHandler.OnDisconnectHandler(Object platformView) in /_/src/Core/src/Handlers/View/ViewHandler.cs:line 221
[mono-rt]    at Microsoft.Maui.Handlers.ElementHandler.DisconnectHandler(Object platformView) in /_/src/Core/src/Handlers/Element/ElementHandler.cs:line 131
[mono-rt]    at Microsoft.Maui.Handlers.ElementHandler.Microsoft.Maui.IElementHandler.DisconnectHandler() in /_/src/Core/src/Handlers/Element/ElementHandler.cs:line 149
[mono-rt]    at Microsoft.Maui.ViewExtensions.DisconnectHandlers(IView view) in /_/src/Core/src/ViewExtensions.cs:line 44
[mono-rt]    at Microsoft.Maui.Controls.Page.SendNavigatedFrom(NavigatedFromEventArgs args, Boolean disconnectHandlers) in /_/src/Controls/src/Core/Page/Page.cs:line 857
[mono-rt]    at Microsoft.Maui.Controls.NavigationPage.SendNavigated(Page previousPage, NavigationType navigationType) in /_/src/Controls/src/Core/NavigationPage/NavigationPage.cs:line 398
[mono-rt]    at Microsoft.Maui.Controls.NavigationPage.MauiNavigationImpl.<>c__DisplayClass7_0.<OnPopAsync>b__2() in /_/src/Controls/src/Core/NavigationPage/NavigationPage.cs:line 820
[mono-rt]    at Microsoft.Maui.Controls.NavigationPage.SendHandlerUpdateAsync(Boolean animated, Action processStackChanges, Action firePostNavigatingEvents, Action fireNavigatedEvents) in /_/src/Controls/src/Core/NavigationPage/NavigationPage.cs:line 700
[mono-rt]    at Microsoft.Maui.Controls.NavigationPage.MauiNavigationImpl.OnPopAsync(Boolean animated) in /_/src/Controls/src/Core/NavigationPage/NavigationPage.cs:line 802
[mono-rt]    at TescoSW.MA.FormsDSH.Base.BaseContentPage.<>c__DisplayClass60_0.<<PopAsync>b__0>d.MoveNext()

[mono-rt] --- End of stack trace from previous location ---
[mono-rt]    at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)
[mono-rt]    at Android.App.SyncContext.<>c__DisplayClass2_0.<Post>b__0() in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/Android.App/SyncContext.cs:line 36
[mono-rt]    at Java.Lang.Thread.RunnableImplementor.Run() in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/Java.Lang/Thread.cs:line 37
[mono-rt]    at Java.Lang.IRunnableInvoker.n_Run(IntPtr jnienv, IntPtr native__this) in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/obj/Release/net9.0/android-35/mcw/Java.Lang.IRunnable.cs:line 59
[mono-rt]    at Android.Runtime.JNINativeWrapper.Wrap_JniMarshal_PP_V(_JniMarshal_PP_V callback, IntPtr jnienv, IntPtr klazz) in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/Android.Runtime/JNINativeWrapper.g.cs:line 22
```

--- a/src/_Shared.Native/Platforms/Android/PointerController.cs
+++ b/src/_Shared.Native/Platforms/Android/PointerController.cs
@@ -68,7 +68,7 @@ internal partial class PointerController : INativePointerController
         _scaleDetector?.Dispose();
         _scaleDetector = null;
 
-        _customScaleListener.Dispose();
+        _customScaleListener?.Dispose();
         _customScaleListener = null!;
     }
 

